composites/Plugin/Shared/tests/__snapshots__/StackedProgresBarTest.js.snap

```diff
@@ -5,16 +5,16 @@ exports[`the BaseButton matches the snapshot 1`] = `
   className="sc-bdVaJa bJbZBf"
 >
   <span
-    className="sc-bwzfXH gpLKCA"
+    className="undefined-part sc-bwzfXH gpLKCA"
```

Comment:
This doesn't seem handy to have in a snapshot. A `className` should be added to the test.